### ISSUE TITLE
Do we want V7.4 to get moved to V10?

### ISSUE THREAD

**Issue by tghosth** [OWASP/ASVS#1946]
Do we want V7.4 to get moved to V10?

**Comment by elarlang** [comment]
Yes. Proposed it here: https://github.com/OWASP/ASVS/issues/1444#issuecomment-1765776826

Even the chapter text says:

> The purpose of error handling is to allow the application to provide security-relevant events for monitoring, triage and escalation. **The purpose is not to create logs**. When logging security-related events, ensure that there is a purpose to the log and that it can be distinguished by SIEM or analysis software.

Reasons to move:

  * it is all about how the program code works and handles errors, and is not directly related to logging
  * it should be a clear message, that security logging IS NOT error logging. As a pentester and trainer, this is something I need to explain too often.

Previously there was no better place than V7, now V10 comes more as a "code quality" paragraph and the current V7.4 suits better there.

But to not cause confusion, I think we need to work with V10 first to be a more suitable place. At the moment it is still "malicious code" by title, but it's not anymore by content.

Another option is to keep it in V7, as all errors in the program code are also security events, but then we need to "re-brand" the V7.

**Comment by tghosth** [comment]
I disagree but not strongly. My reasons:

* I think there is a conceptual fit in V7, made stronger by the fact that it has been there since ASVS v1.
* I worry that V10 is getting overloaded with too much stuff compare dot other chapters.
* I think we can make the split between error logging and security logging clearer in V7.
